spark/src/test/scala/org/apache/comet/CometExpressionCoverageSuite.scala

```diff
@@ -80,62 +80,102 @@ class CometExpressionCoverageSuite extends CometTestBase with AdaptiveSparkPlanH
   // This can be simplified once comet supports MemoryScan, now Comet triggers from the FileScan
   // If MemoryScan supported we can just run Spark examples as is
   val manualTests: Map[String, (String, String)] = Map(
-    "!" -> ("select true a", "select ! true from tbl"),
```

Comment:
compiler warning: Adapting argument list by creating a 2-tuple: this may not be what you want.